user.go

```diff
@@ -17,32 +19,61 @@ type usersRoot struct {
 	Meta  meta   `json:"meta"`
 }
 
+// SocialAccounts are social usernames or urls
+type SocialAccounts struct {
+	GitHub   string `json:"github,omitempty"`
+	LinkedIn string `json:"linkedin,omitempty"`
+	Twitter  string `json:"twitter,omitempty"`
+}
+
 // User represents an Equinix Metal user
 type User struct {
-	ID                    string  `json:"id"`
-	FirstName             string  `json:"first_name,omitempty"`
-	LastName              string  `json:"last_name,omitempty"`
-	FullName              string  `json:"full_name,omitempty"`
-	Email                 string  `json:"email,omitempty"`
-	TwoFactor             string  `json:"two_factor_auth,omitempty"`
+	ID               string          `json:"id"`
+	ShortID          string          `json:"short_id"`
+	FirstName        string          `json:"first_name,omitempty"`
+	LastName         string          `json:"last_name,omitempty"`
+	FullName         string          `json:"full_name,omitempty"`
+	Email            string          `json:"email,omitempty"`
+	SocialAccounts   *SocialAccounts `json:"social_accounts,omitempty"`
+	CustomData       interface{}     `json:"customdata,omitempty"`
+	OptIn            *bool           `json:"opt_in,omitempty"`
+	OptInUpdated     string          `json:"opt_in_updated_at,omitempty"`
+	DefaultProjectID string          `json:"default_project_id,omitempty"`
```

Comment:
default_project_id is nullable, at least for my user (together with some other fields):
```
 "default_project_id": null,
 "language": null,
 "mailing_address": null,
```